src/clr-core/internal/utils/dom.ts

```diff
@@ -17,3 +17,23 @@ export function getElementWidthUnless(element: HTMLElement, unless: boolean) {
   }
   return '';
 }
+
+export type HTMLAttributeTuple = [string, string | boolean];
+
+export function setAttributes(element: HTMLElement, attributeTuples: HTMLAttributeTuple[]) {
```

Comment:
allows us to set attributes on an element like this...

`setAttributes(el, [['title', 'fancy title'], ['style', 'background: #fff'], ['aria-label', 'fancy label']`

Reply:
Oh nice! I didn't think of that and should have. Thanks!

Will update!

Reply:
✅